INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
>done with a lazy check (on/off based on influxdb.enabled)

Lazy check is perfect for what is needed from that `console.log`, thanks.

> At the moment there is a check in messages.server.controller before starting stats, so if the influxdb is disabled, no errors are shown. Can be done after this check if required (in the else). Other errors are logged later.

Hm, should we just hit `messageToInfluxService.save()` even if influxdb isn't configured and do that check later down the logic? Otherwise we'll be adding this check every time we want to write something to Influxdb.